test/test.js

```diff
@@ -292,16 +293,57 @@ describe('onclickoutside hoc', function() {
   });
 
   describe('with child rendering as null', function() {
-    var StatelessComponent = () => null;
```

Comment:
I meant you took a test, and then changed how it worked. That is now how you maintain tests. Instead, copy-paste the test, leave the original alone, rename the copy based on what it's going to test,, and then update its code to do the new thing you're testing for.